RecoTracker/TransientTrackingRecHit/python/TTRHBuilderWithTemplate_cfi.py

```diff
@@ -18,3 +18,7 @@
 # Turn off template reco for phase 2 (when not supported)
 from Configuration.ProcessModifiers.phase2_PixelCPEGeneric_cff import phase2_PixelCPEGeneric
 phase2_PixelCPEGeneric.toModify(TTRHBuilderAngleAndTemplate, PixelCPE = 'PixelCPEGeneric')
+
+# Turn off template reco for phase 2 HLT
+from Configuration.ProcessModifiers.phase2_PixelCPEGeneric4HLT_cff import phase2_PixelCPEGeneric4HLT
+phase2_PixelCPEGeneric4HLT.toModify(TTRHBuilderAngleAndTemplate, PixelCPE = 'PixelCPEGeneric')
```

Comment:
how is this modifier expected to differ from the existing ```phase2_PixelCPEGeneric``` ?

Reply:
As you already pointed out above this modifier differs from the existing one in the fact that this one just switches all the template calls into generic calls, while the existing one also removes any usage of template info (e.g. for the error assignment).

Reply:
Is this the intended behavior?

... anyways, from https://github.com/cms-sw/cmssw/pull/30465#issuecomment-652076405 it sounds like the expected solution for HLT will not use a modifier

Reply:
> Is this the intended behavior?

yes. In the case I provided earlier (in PR https://github.com/cms-sw/cmssw/pull/29018 ) - as you can check in the presentation I gave at the Reco/AT meeting [*],  we don't have (yet) GenErrors for the 3D sensors, so *all* template information must be avoided. Here, instead, @mtosi  just wants to switch Template with Generic CPE (standard configuration)

[*] https://indico.cern.ch/event/893830/contributions/3770697/ 

> ... anyways, from #30465 (comment) it sounds like the expected solution for HLT will not use a modifier

right. I would propose to close this PR.

Reply:
> right. I would propose to close this PR.

@mtosi 
do you agree?

Reply:
> > right. I would propose to close this PR.
> 
> @mtosi
> do you agree?

ping